pwnlib/shellcraft/templates/aarch64/pushstr.asm

```diff
@@ -53,7 +53,6 @@ total = len(pairs) * 16
 
 offset = 0
 %>\
-    /* push ${pretty_string} */
```

Comment:
?

Reply:
in my "tests", this `pretty_string` was very very long, and it hangs my python

Reply:
or better cut this `pretty_string[:1024]`

Reply:
done